delivery_package_type_number_parcels/tests/test_package_type_number_parcels.py

```diff
@@ -0,0 +1,98 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.addons.stock.tests.test_packing import TestPackingCommon
+
+
+class TestStockQuantPackageDelivery(TestPackingCommon):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
```

Comment:
can you maybe use  ? cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))